docarray/array/mixins/io/pushpull.py

```diff
@@ -1,24 +1,183 @@
+import json
 import os
+import os.path
 import warnings
+from collections import Counter
 from pathlib import Path
-from typing import Dict, Type, TYPE_CHECKING, Optional
+from typing import Dict, Type, TYPE_CHECKING, List, Optional, Any
+
+import hubble
+from hubble import Client
```

Comment:
Maybe we should `import Client as HubbleClient` to avoid confusion with the Jina Client that appears in other parts of the code base.

Reply:
done